config/helm/chart/default/templates/Common/kubernetes-monitoring/clusterrole-kubernetes-monitoring-full.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if and .Values.rbac.kubernetesMonitoringFull.create .Values.rbac.activeGate.create .Values.rbac.kspm.create }}
```

Comment:
the condition should be changed to
```
.Values.rbac.kubernetesMonitoringFull.create AND ( .Values.rbac.activeGate.create OR .Values.rbac.kspm.create)
```
according to the ticket.

I set
```
activeGate,create FALSE
kspm.create TRUE
kubernetesMonitoringFull TRUE
```
and new cluster role hasn't been installed

Reply:
It turns out, that KSPM doesn't have any relevance for this at all. It's just about ActiveGate.